### ISSUE TITLE
[CoE Starter Kit - BUG] Admin | Sync Template v3 (Flow Action Details) Error "The response is not in a JSON format."

### ISSUE THREAD

**Issue by AUTigers16** [microsoft/coe-starter-kit#1599]
### Describe the issue

After updating to the December release of the toolkit the **Admin | Sync Template v3 (Flow Action Details)** flow under **Core Components** is failing. 

![image](https://user-images.githubusercontent.com/95765970/145444318-53e344bb-7f59-45c4-bef3-f7fba20c2e23.png)

### Expected Behavior

Flow run succeeds

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.19

### What app or flow are you having the issue with?

Admin | Sync Template v3 (Flow Action Details)

### Steps To Reproduce

This flow runs on its own

**Comment by Jenefer-Monroe** [comment]
Hello I'm afraid we'll need more details in order to assist. Please dive into that failure until you come to the point of failure. 

Open the Apply to each and hit Next failed
![image](https://user-images.githubusercontent.com/89584709/145452558-38c40dce-7190-434e-ab0b-2d99f4766eea.png)

And then continue iterating until you hit the exact point of failure.

**Comment by pbullar** [comment]
Confirmed we are having the same issue

**Comment by Jenefer-Monroe** [comment]
@pbullar similarly, I will need to have you dive down into the failure and return with the exact action that is failing.

**Comment by AUTigers16** [comment]
It is failing at the "List Flows as Admin" Action. It resolved itself in one tenant. I am awaiting to see if it happens in our other tenant. If it resolved there, I will close this thread. It may be working by design.
![image](https://user-images.githubusercontent.com/95765970/145593593-56de7b48-40e5-4bb0-8164-595793feaaeb.png)

**Comment by Jenefer-Monroe** [comment]
One other customer saw a call to List Flows as Admin failing with pagination on for their default environment.  You will need to go to product support for assistance as this repros outside of the toolkit.   

You can use this flow to get a simple repro, import it, select your environment that repros (which you can get by that action you just found), and run.  [Listflowsfailing_20211210142234.zip](https://github.com/microsoft/coe-starter-kit/files/7693349/Listflowsfailing_20211210142234.zip) 

Basically the product is not able to return pages of flows to you for some reason for this environment.  

The person that repro also also saw the repro through the product UX by browsing to the [admin pages](https://admin.powerplatform.microsoft.com/), selecting the environment, and clicking to see the flows in Resources. They saw only a very small list rather than the many flows expected.

**Comment by AUTigers16** [comment]
I was able to check our other tenant and it finally had a successful fun. Its actively going through another run. I believe that one will succeed. Will take note of what you provided above in case the active run fails. I will do ahead and close this report.
